### PR TITLE
README Installation & Usage steps

### PR DESCRIPTION
**Pull request by Thytu**
The README now has :
* An `Installation` step describing how to install the repository.
* An `Usage` step describing how to generate the SDKs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,27 @@
-# unifai-sdk
\ No newline at end of file
+# Gladia's SDKs
+
+Automatic SDK generation from Gladia's OpenAPI definition.
+
+## Installation
+
+```sh
+git clone https://github.com/gladiaio/gladia-sdk.git ; cd gladia-sdk
```
Comment (TomChv) [inline README.md:8]: Explain requirement: which version of python it requires?

File: README.md [hunk 2]
```diff
@@ -1 +1,27 @@
...
+```
+
+Then provide the gladia API URL (by default it's `http://localhost:8080`) to the following command:
+```sh
+cd generator ; python generate_sdk.py $GLADIA_API_URL
+```
+Note that you do not include the `openapi.json` extension within the URL.
+
+
+SDKs will be generated in the [clients directory](clients)
```
Comment (TomChv) [inline README.md:27]: You should add an example so user can ensure everything went good

File: README.md [hunk 3]
```diff
@@ -1 +1,27 @@
...
+## Usage
+
+You first need to start the [Gladia's API](https://github.com/gladiaio/gladia) by following [it README](https://github.com/gladiaio/gladia#magic-start).
+
+We recommend to first clean the  [clients directory](clients) to prevent any deprecated SDK:
+```sh
+rm -rf clients/*
+```
+
+Then provide the gladia API URL (by default it's `http://localhost:8080`) to the following command:
```
Comment (TomChv) [inline README.md:20]: Then set `GLADIA_API_URL` in your environment (by default..)

**Review by TomChv — COMMENTED** [review]
Hey @Thytu, I saw a notification on GitHub so I clicked by curiosity and saw your PR :D

I hope my review will help you :rocket:

**Comment by Thytu** [comment]
> Hey @Thytu, I saw a notification on GitHub so I clicked by curiosity and saw your PR :D
> 
> I hope my review will help you rocket

Hey @TomChv thanks for the feedback 🤘

File: README.md [hunk 4]
```diff
@@ -1 +1,27 @@
...
+
+## Installation
+
+```sh
+git clone https://github.com/gladiaio/gladia-sdk.git ; cd gladia-sdk
+```
+
+## Usage
+
+You first need to start the [Gladia's API](https://github.com/gladiaio/gladia) by following [it README](https://github.com/gladiaio/gladia#magic-start).
```
[the remark this answers is not in the document]
Comment (Thytu) [reply README.md]: fixed in 360ebaeec43c0e715b0e756178187303fe5361c1

File: README.md (hunk 3 above)
Comment (Thytu) [reply README.md]: fixed in 3f6c1d4dbe63ed7b73c3f15d322622de900dd952

File: README.md (hunk 1 above)
Comment (Thytu) [reply README.md:15]: solved in 9e92a7fb60474d5f53c4e22f9299a6ba8b758c36

File: README.md (hunk 2 above)
Comment (Thytu) [reply README.md]: fixed in 17831345715108a0cab9884c44a02439beb5f9e0

**Pull request merged**